src/build.py

```diff
@@ -690,9 +690,11 @@ def AllSources():
                LLVM_GIT_BASE + 'llvm-project.git',
                checkout=RemoteBranch('main')),
         Source('llvm-test-suite', GetSrcDir('llvm-test-suite'),
-               LLVM_GIT_BASE + 'llvm-test-suite.git'),
+               LLVM_GIT_BASE + 'llvm-test-suite.git',
+               checkout=RemoteBranch('main')),
```

Comment:
I guess we could even make 'main' the default now instead of 'master', and then I think it would only be waterfall and wabt that would need `RemoteBranch` (and those we should probably just fix anyway)

Reply:
Yeah I think its time to change the default.. although can can be a followup if you prefer.

Reply:
I just renamed wabt's branch to use `main` :)